docs/source/Components/Webclient.md

```diff
@@ -108,8 +110,16 @@ from the server and display them as inline HTML.
 while the tab is hidden.
 * `oob.js` Defines onSend. Allows the user to test/send Out Of Band json messages to the server.
 * `options.js` Defines most callbacks. Provides a popup-based UI to coordinate options settings with the server.
-* `options2.js` Defines most callbacks. Provides a goldenlayout-based version of the options/settings tab.  Integrates with other plugins via the custom onOptionsUI callback.
+* `options2.js` Defines most callbacks. Provides a goldenlayout-based version of the options/settings tab.
+Integrates with other plugins via the custom onOptionsUI callback.
 * `popups.js` Provides default popups/Dialog UI for other plugins to use.  
+* `text2html.js` Provides a new message handler type: `text2html`, similar to the multimedia and html
+plugins. This plugin provides a way to offload rendering the regular pipe-styled ASCII messages
+to the client.  This allows the server to do less work, while also allowing the client a place to
+customize this conversion process.  To use this plugin you will need to override the current commands
+in Evennia, changing any place where a raw text output message is generated and turn it into a
+`text2html` message. For example: `target.msg("my text")` becomes:
+`target.msg(text2html=("my text", {"type": "tag"}))`
```

Comment:
When reading this, I can think of several questions a casual reader may ask: 
- What HTML does this produce on the client side?
- What is the intent of the `tag` type -- is this the same for all HTML elements or can you provide a type of `span` or `div` separately, for example? 

It would be good with a brief explanation of what this means now that it's mentioned.